pandas/tests/test_common.py

```diff
@@ -222,3 +224,22 @@ def test_standardize_mapping():
 
     dd = collections.defaultdict(list)
     assert isinstance(com.standardize_mapping(dd), partial)
+
+
+@pytest.mark.parametrize('method', ['to_pickle', 'to_json', 'to_csv'])
+def test_compression_size(method, compression):
+
+    df = pd.concat(100 * [DataFrame([[0.123456, 0.234567, 0.567567],
+                                     [12.32112, 123123.2, 321321.2]],
+                                    columns=['X', 'Y', 'Z'])],
+                   ignore_index=True)
+    s = df.iloc[:, 0]
+
+    with tm.ensure_clean() as filename:
+        for obj in [df, s]:
```

Comment:
Can even parametrize the Series and Frame instead of a loop

Reply:
done.